changelogs/unreleased/bump-libcurl-to-7.83.0.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Bump Curl from 7.80.0 to 7.83.0 for static build (gh-6029).
```

Comment:
1. We bundle libcurl in the usual build by default, not only in so called static build (where much more libraries are bundled). So I find 'for static build' here confusing.
2. Please, follow https://www.tarantool.io/en/doc/latest/contributing/release_notes/. I would say 'libcurl is updated <...>'.

Reply:
Updated:
```diff
--- a/changelogs/unreleased/bump-libcurl-to-7.83.0.md
+++ b/changelogs/unreleased/bump-libcurl-to-7.83.0.md
@@ -1,3 +1,3 @@
 ## feature/build
 
-* Bump Curl from 7.80.0 to 7.83.0 for static build (gh-6029).
+* Updated libcurl to version 7.83.0 (gh-6029).
sergeyb@pony:~/sources/MRG/tarantool$ 
```


BTW in changelogs two times used "updated" and five times "Bump":

```
sergeyb@pony:~/sources/MRG/tarantool$ grep -R -i updated changelogs/
changelogs/2.10.0-rc1.md:* Updated libev to version 4.33 (gh-4909).
changelogs/2.10.0-rc1.md:* Updated `libicu` version to 71.1 for static build.
sergeyb@pony:~/sources/MRG/tarantool$ grep -R -i bump changelogs/
changelogs/2.10.0-rc1.md:* Bumped Debian package compatibility level to 10 (gh-5429).
changelogs/2.10.0-rc1.md:* Bumped minimal required debhelper to version 10 (except for Ubuntu Xenial).
changelogs/2.10.0-rc1.md:* Bumped Debian control Standards-Version to 4.5.1 (gh-6390).
changelogs/2.10.0-rc1.md:* Bumped OpenSSL from 1.1.1f to 1.1.1n for static build (gh-6947).
changelogs/2.10.0-rc1.md:* Bumped Debian packages tarantool-common dependency to use luarocks 3 (gh-5429).
```